organizers/2023-07-13.html

```diff
@@ -0,0 +1,144 @@
+---
+layout: organizers
+---
+
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>July 13th, 2023</h2>
+    <img data-src="/img/django-logo-white-transparent.svg"
+         style="width: 40%; background: none;"
+         alt="Django logo"
+         class="plain">
+    <h2>Talks starting at 19:00!</h2>
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of The London Django Meetup Group is to provide a fun,
+        welcoming and professional environment for a diverse group of people.
+    </p>
+    <p>
+        Code of conduct:
+        <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    </p>
+</section>
+
+<section>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Kraken Tech (part of the Octopus Energy Group)</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section>
+    <img data-src="/img/2022_KrakenV2_stacked_White.png"
+         style="width: 60%;"
+         alt="Kraken Tech"
+         class="plain">
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25 million households worldwide.
+        </em>
+    </p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News</h2>
+    <h2>👩🏻‍💻 Talks</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Django security releases</h2>
+    <p>July 3</p>
+    <p>4.2.3, 4.1.0, and 3.2.20</p>
+    <p>Potential regular expression denial of service vulnerability in <code>EmailValidator</code>/<code>URLValidator</code></p>
+    <p><em>Don’t delay, upgrade today!</em></p>
+</section>
+
```

Comment:
Let's promote Django News!

```suggestion
<section>
    <h1>Django News</h1>
    <p>Weekly newsletter of blog posts, developments, talks, packages, etc.</p>
    <p>django-news.com</p>
</section>    

<section>
    <h1>🐍 News</h1>
    <h2>Django security releases</h2>
    <p>July 3</p>
    <p>4.2.3, 4.1.0, and 3.2.20</p>
    <p>Potential regular expression denial of service vulnerability in <code>EmailValidator</code>/<code>URLValidator</code></p>
    <p><em>Don’t delay, upgrade today!</em></p>
</section>

```